src/api.test.ts

```diff
@@ -9,13 +7,11 @@ import { BadRequestError, CURRENT_API_VERSION, ExasolExtension, InternalServerEr
 async function readPackageJson(): Promise<any> {
     const fileContentBuffer = await readFile("package.json")
     const fileContent = fileContentBuffer.toString("utf8")
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Do I understand correctly, that lint disable could be removed after adding the dependency to typescript-eslint?

Reply:
Not sure. I assumed that the upgrade either removed the rule or improved to so that it doesn't detect a false positive here ;)